frontend_tests/node_tests/index.js

```diff
@@ -80,6 +80,19 @@ global.use_template = function (name) {
     Handlebars.templates[name] = Handlebars.compile(data);
 };
 
+function stylesheets() {
+    // TODO: Automatically get all relevant styles.
+    var data = '';
+    data += '<link href="../../static/styles/fonts.css" rel="stylesheet">\n';
+    data += '<link href="../../static/styles/portico.css" rel="stylesheet">\n';
+    data += '<link href="../../static/styles/thirdparty-fonts.css" rel="stylesheet">\n';
+    data += '<link href="../../static/styles/zulip.css" rel="stylesheet">\n';
+    data += '<link href="../../static/third/bootstrap/css/bootstrap.css" rel="stylesheet">\n';
+    data += '<link href="../../static/third/bootstrap-notify/css/bootstrap-notify.css" rel="stylesheet">\n';
```

Comment:
We should probably add a comment next to the `STYLESHEETS` definition in settings.py noting that we need to keep this up to date.